cloudigrade/util/tests/redhatcloud/test_sources.py

```diff
@@ -112,6 +112,30 @@ def test_get_sources_cloudigrade_application_type_success(self, mock_get):
         self.assertEqual(response_app_type_id, cloudigrade_app_type_id)
         mock_get.assert_called()
 
+    @patch("requests.get")
+    def test_get_sources_cloudigrade_application_type_is_cached(self, mock_get):
+        """Assert get_cloudigrade_application_type_id returns cached id."""
+        cloudigrade_app_type_id = _faker.pyint()
+        expected = {"data": [{"id": cloudigrade_app_type_id}]}
+        mock_get.return_value.status_code = http.HTTPStatus.OK
+        mock_get.return_value.json.return_value = expected
+
+        response_app_type_id = sources.get_cloudigrade_application_type_id(
+            self.account_number
+        )
+        self.assertEqual(response_app_type_id, cloudigrade_app_type_id)
+
+        """Call it again and make sure the original, cached value is returened"""
+        not_cloudigrade_app_type_id = _faker.pyint()
+        not_expected = {"data": [{"id": not_cloudigrade_app_type_id}]}
+        mock_get.return_value.json.return_value = not_expected
+
+        response_app_type_id = sources.get_cloudigrade_application_type_id(
+            self.account_number
+        )
+        self.assertEqual(response_app_type_id, cloudigrade_app_type_id)
+        mock_get.assert_called_once()
```

Comment:
@abellotti I don't think there's a convenient way to use `assert_called_once_with` because the argument is the request being made to Sources and is built in the `get_cloudigrade_application_type_id` function